server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -308,7 +308,7 @@ def main():
             report.write("{}.{}({})\n".format(prog, config.TS, config.PBENCH_ENV))
 
             try:
-                # Check the data integrity in ARCHIVE (Question 1).
+                # Check the data integrity in ARCHIVE.
```

Comment:
Why remove "Question 1"?  These are here because the algorithm described at the top of the module asked those questions, and the comment records where that question is answered.

If you don't want them, can you find another way to connect the module comment to the code?